ci/local-run-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
How would we use this script? Could/Should we add a tiny section in the README explaining why/how to use this? 🙂

Reply:
Yes good point it's not very telling, since this is only for local development and not intended for customer I think it's fine to not document it directly in the readme, I'd rather add a bit of context in the script itself and maybe add a `package.json` script for it.